ejb/WFLY-11622_Multiple_delivery_group_for_MDBs.adoc

```diff
@@ -0,0 +1,65 @@
+= WFLY-11622 Multiple Delivery Group Support for Message Driven Beans
+:author:            Flavia Rainone
+:email:             frainone@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently an MDB can belong to a delivery-group, a group identified by a String name.
+If the MDB belongs to a delivery-group, delivery of messages for that MDB will be enabled only when the delivery-group is active.
+If the MDB belongs to a delivery-group and is clustered singleton, delivery wlil be enabled only when the delivery-group is active in the singleton node of the cluster.
+This REF will enhance delivery-group capability, by making possible to associate an MDB to more than one delivery-groups at once.
+Delivery will be enabled only when all delivery-groups the MDB belongs to are active. In the clustered singleton case, that implies that delivery is active only 
+in the singleton node of the cluster and only if all delivery-groups associated with the MDB are active.
+
```

Comment:
@fl4via I'm not sure for which customer use case is this feature good for? Do we have any examples?

Reply:
@mnovak1 the only info I have is at Jira, and apart from that, the best example is at quickstart, there you will be able to see the delivery groups working in conjunction with cluster and you will be able to activate and deactivate a delivery group, seeing in action the delivery of messages being suspended and resumed, I'll update document asap with all this information